src/core/src/bootstrap/Constants.py

```diff
@@ -112,6 +112,13 @@ class AutomaticOSPatchStates(EnumBackport):
         DISABLED = "Disabled"
         ENABLED = "Enabled"
 
+    # List of auto OS update services in Yum
+    # todo: Q for reviewer: Should this list be in YumPackageManager, since it is only used there?
```

Comment:
yes, to prevent constants exploding, and potentially remaining as dead code later if implementations change

Reply:
Will have to look into how to declare constants the packagemanager and refer it, added a note to address this in the next PR